Sources/VitaminUIKit/Components/Chip/README.md

```diff
@@ -204,9 +200,9 @@ chip.enabled = true
 You can provide an action that will be triggered when you tap on the chip.
 This action is a closure, that takes the tapped `VitaminChip` as only parameter.
 
-This is settable through the `action` property, or the or the `action` constructor parameter.
+This is settable through the `action` property, or the `action` constructor parameter.
 
-Typcally, it is here than you have to handle the change of state for other single choice chips, or the dismissal of input chips.
+Typically, this is the place where you have to handle the change of state for other single choice chips (sonce only one single choice chip can be selected at a time)
```

Comment:
Not sure about the suggestion 🤔 
```suggestion
Typically, this is the place where you have to handle the change of state for other single choice chips (since only one single choice chip can be selected at a time)
```